File: DSINTERFAZ/ListViewDemo/app/src/main/java/com/iesvirgendelcarmen/dam/listviewdemo/Persona.java
package com.iesvirgendelcarmen.dam.listviewdemo;

/**
 * Created by matinal on 16/10/17.
 */

public class Persona{

    private String nombre;
    private String apellidos;

    public Persona(String nombre, String apellidos) {
        this.nombre = nombre;
        this.apellidos = apellidos;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    @Override
    public String toString() {
        return
                "nombre '" + nombre + '\'' +
                ", apellidos '" + apellidos + '\''
                ;
    }
}

File: DSINTERFAZ/TypeScript/Ejer2/persona.js
"use strict";
exports.__esModule = true;
var Persona = /** @class */ (function () {
    function Persona(nombre, apellidos, salario) {
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.salario = salario;
    }
    Persona.prototype.setNombre = function (nombre) {
        this.nombre = nombre;
    };
    Persona.prototype.getNombre = function () {
        return this.nombre;
    };
    Persona.prototype.setApellidos = function (apellidos) {
        this.apellidos = apellidos;
    };
    Persona.prototype.getApellidos = function () {
        return this.apellidos;
    };
    Persona.prototype.setSalario = function (salario) {
        this.salario = salario;
    };
    Persona.getSalarioTotal = function (trabajadores) {
        var total = 0;
        for (var _i = 0, trabajadores_1 = trabajadores; _i < trabajadores_1.length; _i++) {
            var emp = trabajadores_1[_i];
            total += emp.getSalario();
        }
        return total;
    };
    Persona.prototype.getSalario = function () {
        return this.salario;
    };
    Persona.prototype.toString = function () {
        return "Nombre: " + this.nombre + "Apellidos " + this.apellidos
            + "Salario " + this.salario;
    };
    ;
    return Persona;
}());
exports.Persona = Persona;
